Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN apt-get update && apt-get install -y \
     binutils gdal-bin libproj-dev git \
     && rm -rf /var/lib/apt/lists/*
 COPY --from=builder $PYSETUP_PATH $PYSETUP_PATH
-COPY --from=builder-node $NPMSETUP_PATH $NPMSETUP_PATH
+COPY --from=builder-node $NPMSETUP_PATH cobrand_hackney/
```

Comment:
Should this come after the WORKDIR line? How is it knowing where to copy it otherwise? (Sorry for basic questions, just making sure I understand)

Reply:
Yes, you are right it should, good catch! (I didn't pick it up as I'd forgotten I'd built the modules locally once so it all looked correct when I stood up my environment.) I'll push a fixup.